### PR TITLE
Move CLI from `__main__.py` to `entrypoint.py`

### LINKED ISSUE #161: Move __main__.py to entrypoint.py
pytest (since a few versions) ignores doctests in `__main__.py` so we should move the contents of this file elsewhere and just call it from the actual `__main__.py`.

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Checklist
* [x] Added an entry in `doc/news/`. 
* [x] Added a test for this change. (not required)
* [x] Updated the documentation. (not required)

Changes the entry point such that the doctests of the CLI are no longer ignored during testing.

Fixes #161 

Dependencies:
- [x] #169 
- [x] #175

### CODE REVIEW

**Comment by DunklesArchipel** [comment]
not sure is `__main__.py`  and `entrypoint.py` should share the same doctest at the beginning of the document.

File: .github/workflows/doc.yml [hunk 1]
```diff
@@ -43,7 +47,7 @@ jobs:
         run: |
           python -m http.server 8880 --directory doc/generated/html &
           sleep 1
-          linkchecker --check-extern http://localhost:8880/svgdigitizer/
+          linkchecker --check-extern http://localhost:8880/svgdigitizer/ --ignore-url="https://repology.org/*"
```
Comment (saraedum) [inline .github/workflows/doc.yml:50]: Could you add a comment in the workflow why you want to exclude repology?

Comment (DunklesArchipel) [reply .github/workflows/doc.yml:51]: This change was added in #169 since the check passes locally but not in the GitHub workflow.

Comment (DunklesArchipel) [reply .github/workflows/doc.yml:51]: A comment was added.

Comment (DunklesArchipel) [reply .github/workflows/doc.yml:51]: is this the only change you wanted and would it be ready for merging??

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
@DunklesArchipel please merge if you agree with the changes here.

**Pull request merged**